src/Form/RemoteQuestionForm.php

```diff
@@ -28,6 +28,11 @@ public function form(array $form, FormStateInterface $form_state) {
       }
     }
 
+    $form['langcode'] = [
```

Comment:
I think adding this as a comment could make it more clear for someone just reading this file in particular:
```suggestion
    // Need to set language manually because Question is not marked as 'translatable'.
    $form['langcode'] = [
```